packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -169,12 +171,15 @@ export const VSelect = genericComponent<new <
           modelValue={ model.value.map(v => v.props.value).join(', ') }
           onUpdate:modelValue={ v => { if (v == null) model.value = [] } }
           validationValue={ props.modelValue }
+          placeholder={ props.placeholder }
+          persistentPlaceholder={ props.persistentPlaceholder }
           class={[
             'v-select',
             {
               'v-select--active-menu': menu.value,
               'v-select--chips': !!props.chips,
               [`v-select--${props.multiple ? 'multiple' : 'single'}`]: true,
+              'v-select--selected': !!props.modelValue,
```

Comment:
```suggestion
              'v-select--selected': mode.value.length,
```

Reply:
I tested with `model.value.length` and the result is always true.  But I tested with `props.modelValue.length` and it behaves normally, should I use this property?
```vue
   'v-select--selected': props.modelValue.length,
```